zio-http/src/main/scala/zhttp/service/server/CompressionOptions.scala

```diff
@@ -0,0 +1,34 @@
+package zhttp.service.server
+
+import io.netty.handler.codec.compression.{CompressionOptions => JCompressionOptions, StandardCompressionOptions}
+
+final case class CompressionOptions(level: Int, bits: Int, mem: Int, kind: CompressionOptions.CompressionType) { self =>
```

Comment:
I think we should move this to `ServerConfig` companion object, if that config is the only one that needs it.